packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -36,11 +36,14 @@ module.exports = (themeOptions = {}) => {
   const pluginOptions = { ...defaultOptions, ...themeOptions };
   validateThemeOptions(pluginOptions);
 
+  const productionHostname = 'docs.commercetools.com';
+
   return {
     siteMetadata: {
       author: 'commercetools',
-      productionHostname: 'docs.commercetools.com',
+      productionHostname,
       betaLink: null,
+      siteUrl: `https://${productionHostname}/${pluginOptions.websiteKey}`,
```

Comment:
Note: the `websiteKey` is not a guarantee that it's the same as the URL path.

What you're looking for is the `pathPrefix` I think.